x-pack/plugins/fleet/public/applications/integrations/sections/epm/components/package_card.tsx

```diff
@@ -90,30 +91,36 @@ export function PackageCard({
 
   const testid = `integration-card:${id}`;
   return (
-    <TrackApplicationView viewId={testid}>
-      <Card
-        data-test-subj={testid}
-        layout="horizontal"
-        title={title || ''}
-        titleSize="xs"
-        description={description}
-        hasBorder
-        icon={
-          <CardIcon
-            icons={icons}
-            packageName={name}
-            integrationName={integration}
-            version={version}
-            size="xl"
-          />
-        }
-        onClick={onCardClick}
-      >
-        <EuiFlexGroup gutterSize="xs">
-          {verifiedBadge}
-          {releaseBadge}
-        </EuiFlexGroup>
-      </Card>
-    </TrackApplicationView>
+    <WithGuidedOnboardingTour
+      packageKey={name}
+      isTourVisible={isGuidedOnboardingActive}
+      tourType={'integrationCard'}
```

Comment:
```suggestion
      tourType={'integrationCard'}
      tourOffset={10}
```
Can we add an offset so that the beacon is better aligned on the card?